### PR TITLE
Update docs for XRPFees amendment

### PR DESCRIPTION
**Pull request by oeggert**
Fields have been updated for the `SetFee` pseudo-transaction and `FeeSettings` ledger object. Updating corresponding reference docs.

To be merged with 1.10.0 stable release.

### CODE REVIEW

**Comment by intelliot** [comment]
@ximinez

File: content/references/protocol-reference/ledger-data/ledger-object-types/feesettings.md [hunk 1]
```diff
@@ -41,6 +41,18 @@ The `FeeSettings` object has the following fields:
 
 **Warning:** The JSON format for this ledger object type is unusual. The `BaseFee`, `ReserveBase`, and `ReserveIncrement` indicate drops of XRP but ***not*** in the usual format for [specifying XRP][Currency Amount].
 
+
+If the _[XRPFees amendment][]_ is enabled, the `FeeSettings` object has these fields instead:
+
+| Name                    | JSON Type | [Internal Type][] | Required? | Description            |
+|:------------------------|:----------|:------------------|:----------|:-----------------------|
+| `BaseFeeDrops`          | String    | Amount            | Yes       | The [transaction cost](transaction-cost.html) of the "reference transaction" in drops of XRP. |
+| `Flags`                 | Number    | UInt32            | Yes       | A bit-map of boolean flags enabled for this object. Currently, the protocol defines no flags for `FeeSettings` objects. The value is always `0`. |
```
Comment (DennisDawson) [inline content/references/protocol-reference/ledger-data/ledger-object-types/feesettings.md]: bitmap (no hyphen)

**Review by DennisDawson — APPROVED** [review]
One suggestion, otherwise good to go.

**Comment by oeggert** [comment]
@ximinez I'm going to merge this for now, since I'm fairly certain the changes are fine. If you find anything wrong I'll fix it in a separate PR.

**Review by alaslyalasel — COMMENTED** [review]
alasly alasyl MTANB

**Comment by oeggert** [comment]
@alaslyalasel Could you clarify what MTANB means? I'm not familiar with this acronym.

**Pull request merged**